stellar-contract-env-common/src/raw_val.rs

```diff
@@ -42,7 +42,7 @@ pub enum Tag {
 }
 
 #[repr(transparent)]
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Default)]
 pub struct RawVal(u64);
```

Comment:
A default value for RawVal seems suspicious. The default value will be a zero u63. It might be advantageous to make it a void. Could we provide a Default impl that makes the default void?

Reply:
Sure I guess. This is only to support `TinyVec` but .. yeah I can set it to void.

Reply:
(fixed)